LICENSE

```diff
@@ -0,0 +1,21 @@
+    MIT License
+
+    Copyright (c) Microsoft Corporation and other contributors.
```

Comment:
is this really a Microsoft Corp project or should we use `stac-utils` 🤷‍♂️ 

just looking at all other projects in stac-utils org, we have a tons of differents `org`

Reply:
stac-utils isn't a copyright holder; see above comment. Standard practice (in my experience) is to list the original holder and say others, or list copyright holders specifically. Technically every contributor retains the copyright of their contributions (or the companies who were paying for that code to be developed). Copyright doesn't get signed over to anyone else unless there's a CLA that explicitly states that (like the Apache CLA, I believe)